packages/react/spec/auto/inputs/PolarisAutoTextInput.spec.tsx

```diff
@@ -189,7 +189,7 @@ describe("PolarisAutoTextInput", () => {
     test("it renders number inputs with any step value when no decimals are set", async () => {
       result = render(<PolarisAutoNumberInput field="inventoryCount" />, getCreateWrapper());
       const field = result.container.querySelector(`input[name="widget.inventoryCount"]`);
-      expect(field).toHaveAttribute("step", "any");
+      expect(field).toHaveAttribute("step", "1");
```

Comment:
can you add a test here for a field that has a step of a decimal value?

Reply:
Immediately below this test there's a test with a field config decimal value of 2